src/api/firebase.js

```diff
@@ -222,3 +222,39 @@ export async function deleteItem() {
 	 * this function must accept!
 	 */
 }
+
+export function comparePurchaseUrgency(data) {
+	const now = new Date();
+	const newData = data.map((item) => {
+		const daysBeforePurchase = getDaysBetweenDates(
+			item.dateNextPurchased.toDate(),
+			now,
+		);
+
+		item.daysBeforePurchase = daysBeforePurchase;
+
+		if (daysBeforePurchase < -60) {
```

Comment:
What is the purpose of this number (-60)? 
These constant values must be defined with appropriate names.